src/components/molecules/ProfileFeed.js

```diff
@@ -0,0 +1,68 @@
+import React, {Component} from "react";
+
+import Card from "./Card";
+import config from "../../config";
+
+class ProfileFeed extends Component {
+    state = {
+        articles: []
+    };
+
+    constructor(props) {
+        super(props);
+
+        this.user = props.user;
```

Comment:
Upon testing locally, it seems as though props are passed to the `render()` function. This would make the constructor here redundant - same for the other classes